apps/settings/locales/settings.en-US.properties

```diff
@@ -1233,7 +1233,8 @@ hud-security=Security Issues
 hud-performance=Performance
 hud-reflows=Reflows
 hud-jank=Jank
-jank-threshold=Jank Threshold
+jank-threshold=Jank Threshold (ms)
+reflow-threshold=Duration Threshold (ms)
```

Comment:
If you follow my above suggestion, please change this to `hud-reflow-threshold` as well.